### PR TITLE
[SWARM-1769] Adding a basic keycloak-server test

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

File: testsuite/testsuite-keycloak-server/src/test/java/org/wildfly/swarm/keycloak/server/KeycloakServerArquillianTest.java [hunk 1]
```diff
@@ -32,16 +37,21 @@
...
 
     @Deployment(testable = false)
-    public static Archive createDeployment() {
+    public static Archive<?> createDeployment() {
         JARArchive deployment = ShrinkWrap.create(JARArchive.class);
         deployment.add(EmptyAsset.INSTANCE, "nothing");
         return deployment;
     }
 
+    @CreateSwarm
```
Comment (kenfinnigan) [inline testsuite/testsuite-keycloak-server/src/test/java/org/wildfly/swarm/keycloak/server/KeycloakServerArquillianTest.java]: This particular method shouldn't be necessary, as that should be the default behavior of the container

Comment (sberyozkin) [reply testsuite/testsuite-keycloak-server/src/test/java/org/wildfly/swarm/keycloak/server/KeycloakServerArquillianTest.java]: That is done... The actual endpoint address is used to access the welcome page, so getting 200 on that should be sufficient to confirm the server is up; pinging other endpoints require a more involved set up (which is what the keycloak test now does)

**Comment by sberyozkin** [comment]
Linux one seems to be hanging here...

**Comment by sberyozkin** [comment]
retest this please

**Comment by kenfinnigan** [comment]
retest this please

**Review by Ladicek — APPROVED** [review]

**Pull request merged**